source/develop/release-management/features/network/editable-network-names.html.md

```diff
@@ -0,0 +1,71 @@
+
+This RFE proposes 2 principal changes to this functionality that will allow network names to be modifiable even while
+attached to hosts and VM's vNIC:
+
+1) Remove the unnecessary blocking of renaming networks while only having vNIC profiles attached to VM's.
+
+2) Upon network creation, let users decide whether they'd like the network's name to be editable:
+    - If editable, the network's VDSM name is set to be dervied from the network's GUID (onXXX...) regardless
+      of whether the network's name could be used as VDSM name. This allows the name to be edited without any
+      changes to VDSM name on the host side.
+    - If not editable, current behavior remains. VDSM name may or may not equal the network name, and further changes
+      while attached to hosts are prohibited as they may require changes to VDSM name.
+
+3) Modification of "nameIsEditable" state of existing networks:
+    - Editable to non-editable is prohibited
+    - Non-editable to editable is allowed
+
+## Benefit to oVirt
+
+Users will be able to modify network names even while attached to hosts and vNIC's. This alleviates the users from the need to detach networks from hosts/VMs just for the sake of modifying network names. 
+
+
+## Suggested Solution
+
+Stage 1, allow changes to network names while only attached to vNICs:
+There is currently no reason to disallow changing network names of networks that are merely used in vNICS (networks which aren't attached to hosts but whose vNIC profiles are used in VMS), yet is nonetheless blocked.
+Allow network name changes to such networks, and modify the network's default vNIC profile accordingly.
+
+Stage 2, backend support for network creation:
+A new boolean (a la "nameIsEditable") will be introduced in network's business entity. Upon network creation, the parameter will be set
+based on user preference. The parameter will affect VDSM name generation; if it is set to true, VDSM name is set based on GUID regardless
+whether the network name is can be used as a bridge name. if it is set to false, VDSM name is generated as usual.
+
+Current restriction to name changes should be lifted, and instead a new validation should be introduced that takes "nameIsEditable" under consideration.
+
+REST support is introduced, with the new boolean set to false by default due to backwards considerations and with the boolean reported in queries. VDSM name should be reported as well.
+
+Stage 3, UI:
+VDSM name is currently only displayed on host side. "On host identifier" column should be added in networks service and displayed in networks' info.
+
+"Editable Name" checkbox is added in the network creation dialog. It should be displayed and be grayed out in the network edit dialog.
+
+Stage 4, Editable state modification:
+The "Editable Name" checkbox should no longer be grayed out and should be operational.
```

Comment:
Hi
I find this sentence a bit confusing now, because if:
  Editable to non-editable is prohibited then the "Editable Name" checkbox should be grayed out.
And optional(not grayed out) to new networks/not editable yet networks.